### ISSUE TITLE
How to convert the output of Percolator by poutmzid?

### ISSUE THREAD

**Issue by plantton** [percolator/pout2mzid#5]
I'm wondering if this tool is still active, and can be applied to the latest version of Percolator, to convert the output of Percolator into mzIdentML format?

**Comment by percolator** [comment]
Yes, the pout2mzid is still maintained. Please let me know if you have any
problems running the tool.

**Comment by tivdnbos** [comment]
Since this issue was stil open, I thought I could ask it here.
I was wondering if it's also possible to go from from a percolator output file in tab separated format to an mzid file? Or, does it has to be already in xml format?

**Comment by percolator** [comment]
Unfortunately, it has to be given as a percolator pout.xml file.
--Lukas

**Comment by tivdnbos** [comment]
Thanks for the quick reply, Lukas! 
Regarding that, does it require to have a pin file in XML format, or can that one still be in tab separated format? So, that I rerun percolator with as pin file a tab separated file, and ask to create a pout.xml file?

**Comment by percolator** [comment]
Yes that should work, percolator can read tab delimited files and
write pout XML files.

**Comment by tivdnbos** [comment]
Thanks! 
I've been able to create .pout.xml files given a tsv pin file, but when running
`pout2mzid -p <my_pout>.pout.xml` 
I receive the error "Segmentation fault (core dumped)". Did you receive this error message before?

**Comment by percolator** [comment]
That sounds bad. What input file did you use?

**Comment by tivdnbos** [comment]
I attached here one of the pout.xml files. To be able to upload this on github, I changed the extension from .pout.xml to .txt.

[151222_NJ_Felipe_Y11_allfeatures.txt](https://github.com/percolator/pout2mzid/files/6001675/151222_NJ_Felipe_Y11_allfeatures.txt)

**Comment by percolator** [comment]
No, I cannot recreate the seg.fault. However, as I do not have the mzid
file you try to patch, I can only see the first part of the processing.
